bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/validation/zeebe/SequenceFlowValidator.java

```diff
@@ -39,5 +41,25 @@ public void validate(
         validationResultCollector.addError(0, "Must have a condition or be default flow");
       }
     }
+
+    if (element.getSource() instanceof InclusiveGateway) {
+      final InclusiveGateway gateway = (InclusiveGateway) element.getSource();
+      if (gateway.getOutgoing().size() > 1) {
```

Comment:
Hi,@korthout,if we set this validator, it will affect the join inclusive gateway which connect to a end event, do we must have a condition for this sequence flow?

Reply:
@saig0 I'd like your opinion on this.

If we look at exclusive gateway, the spec is clearly stating that each sequence flow must either have a condition or be the default: BPMN Table 13.2, but we seem to have taken a different approach and allow no condition on non-default outgoing sequence flow if it's the only outgoing sequence flow.
> In case all conditions evaluate to false and a default flow has not been
specified, an exception is thrown.

❓ Should we allow the same for inclusive gateway or should we stick to the spec?
<img width="487" alt="Screen Shot 2022-08-05 at 08 43 12" src="https://user-images.githubusercontent.com/3511026/183017604-845d2203-eb4d-42fc-86a0-642f788c47fa.png">

Reply:
@korthout we should follow the same approach as the exclusive gateway. It makes sense to define no condition on an inclusive gateway that is used to join multiple incoming paths and has only one outgoing path.

I think this doesn't violate the BPMN spec. In the spec, it differs between diverging and converging sequence flows.

![image](https://user-images.githubusercontent.com/4305769/183025605-6dadabd7-bebf-4fe0-8485-a21e87c82f40.png)

![image](https://user-images.githubusercontent.com/4305769/183025741-cc28bc9d-70df-4bd9-8d6b-ad5bcfa1c935.png)

(chapter "8.3.9 Gateways" - page 90)

Reply:
@saig0 OK. To summarize, when we would support the converging (i.e. joining) behavior, then it makes sense to allow a single outgoing sequence flow without condition. While we don't yet support the converging behavior it doesn't really matter whether we allow this.

@skayliu No need to change this. Sorry for the confusion.